scripts/src/admin/translations.ts

```diff
@@ -0,0 +1,162 @@
+import { ExportToCsv, Options as ExportCsvOptions } from "export-to-csv";
+import { FindManyOptions } from "typeorm";
+
+import * as fs from "fs";
+
+import { ContentType, Offer, OfferContent } from "../models/offers";
+
+type CsvRow = {
+	Type: string;
+	Key: string;
+	Default: string;
+	Translation: "";
+	"Character limit": number;
+};
+
+type OfferContentContent = {  // I know, but I didn't have a better name for the 'content' field in the OfferContent table
+	pages?: any[]
+	confirmation?: {
```

Comment:
I'm not used to the mobile pr ui and am resolving your comments by mistake. We don't need it at the moment but since I've added it already I prefer to leave it there since we will want to translate spend offers soon or later and then they'll be supported already.